st2common/st2common/services/workflows.py

```diff
@@ -541,9 +541,12 @@ def request_action_execution(wf_ex_db, task_ex_db, st2_ctx, ac_ex_req, delay=Non
     # Identify the runner for the action.
     runner_type_db = action_utils.get_runnertype_by_name(action_db.runner_type['name'])
 
+    # Identify action pack name
+    pack_name = action_ref.split('.')[0] if action_ref else st2_ctx.get('pack')
```

Comment:
Orquesta task can be action less. Example at https://github.com/StackStorm/st2/blob/master/contrib/examples/actions/workflows/tests/orquesta-test-action-less-tasks.yaml#L12.  Please add a use case for this line that shows it can handle task that is action less.

Reply:
Tested of action less workflow `examples.orquesta-test-action-less-tasks`  and it went well. Not sure about `add a use case`, is to add comment or unit test case?

Reply:
Ok. Then we're good.  Just want to make sure this use case is covered.  Thank you for your diligence.  It's appreciated.